### PR TITLE
Add appengine creation check

### PR DESCRIPTION
**Pull request by hacktobeer**
Add check to see if AppEngine is already enabled as a service in the project.

#4

### CODE REVIEW

**Comment by hacktobeer** [comment]
@berggren ptal

File: deploy.sh [hunk 1]
```diff
@@ -30,8 +30,11 @@ if [[ "$*" != *--use-gcloud-auth* ]] ; then
   SA_NAME="terraform"
   SA_MEMBER="serviceAccount:$SA_NAME@$DEVSHELL_PROJECT_ID.iam.gserviceaccount.com"
 
-  # Create AppEngine app in order to activate datastore
-  gcloud app create --region=us-central
+  # Create AppEngine app, if not already exists, in order to activate datastore
+  if ! gcloud services list | grep appengine
+  then
+    gcloud app create --region=us-central
+  fi
```
Comment (berggren) [inline deploy.sh:37]:
```suggestion
  if ! gcloud services list | grep appengine; then
    gcloud app create --region=us-central
  fi
```

Comment (berggren) [reply deploy.sh:37]: just a small nit - to stay consistent with the rest of the file.

**Review by berggren — APPROVED** [review]
Just a small nit, otherwise LGTM

File: deploy.sh (hunk 1 above)
Comment (hacktobeer) [reply deploy.sh]: done

**Pull request merged**